### ISSUE TITLE
CC-by-sa-4.0 appears to be inadequate

### ISSUE THREAD

**Issue by softwaresinc** [hermitdave/FrequencyWords#19]
All of the terms of the CC licence revolve around "copy and redistribute the material"
There is nothing that grants anyone the ability to simply use the "material" (i.e. the word data).
For example, store a word list in a database in order to make decisions based on the popularity of words entered by a user.
In that case the data is not actually shared. Some aspect of a display might change, but the "material" is not shared.

**Comment by hugolpz** [comment]
@softwaresinc you just have to add credits somewhere in your project.

**Comment by tanguylegazon** [comment]
> [@softwaresinc](https://github.com/softwaresinc) you just have to add credits somewhere in your project.

Hi @hugolpz,

Just sharing my view: mere attribution doesn’t seem enough with CC BY-SA 4.0. Its copyleft nature means that any project incorporating the word lists would have to adopt the same license, which feels quite restrictive.

Perhaps @hermitdave could offer his perspective on this and clarify what he intends with the project license.

**Comment by softwaresinc** [comment]
Agreed. The term "build upon" under "Sharealike" can be interpreted to force a massive project under CC BY-SA 4.0 when it incorporates even a tiny part of this source.

**Comment by hugolpz** [comment]
You could go for CC-BY then, which is more flexible. Or even CC0 / Public Domain, with a one liner to encourage crediting. Something like :

> Credit : [CC0 Hermite Dave](https://github.com/hermitdave/FrequencyWord)

Most reusers will be follow, copy-paste and cite you.
